topcoffea/params/xsec.yml

```diff
@@ -131,30 +131,49 @@ tWll : 0.01123
 VHnobb : 0.952
 SSWW   : 0.02794
 
-########################
-# Run3 Values (13p6 TeV)
-########################
+#########################
+# Run3 Values (13p6 TeV)#
+#########################
+
+#Numbers from TOP-22-012
+TWminus_DR_AtLeastOneLepton_13p6TeV    : 24.2
+TbarWplus_DR_AtLeastOneLepton_13p6TeV  : 24.2
 
 #Numbers from SMP-24-001
-DYto2L-2Jets_MLL-50_13p6TeV    : 6345.99
-WtoLNu-2Jets_13p6TeV           : 64481.58
-WW_13p6TeV                     : 125.7
-TTto2L2Nu_13p6TeV              : 98.0
-TbarWplusto2L2Nu_13p6TeV               : 4.67
-TWminusto2L2Nu_13p6TeV                 : 4.67
-TTHtoNon2B_13p6TeV                     : 0.24111
-
-#https://twiki.cern.ch/twiki/bin/viewauth/CMS/HiggsZZ4lRunIII
-GluGluToContinto2Zto2E2Mu_13p6TeV      : 0.00624157
-GluGluToContinto2Zto2E2Tau_13p6TeV     : 0.00624157
-GluGluToContinto2Zto2Mu2Tau_13p6TeV    : 0.00624157
-GluGlutoContinto2Zto4Mu_13p6TeV        : 0.00303575
-GluGlutoContinto2Zto4E_13p6TeV         : 0.00305851
-GluGlutoContinto2Zto4Tau_13p6TeV       : 0.00303575
+DYto2L-2Jets_MLL-50_13p6TeV        : 6345.99
+DYto2L-2Jets_MLL-10to50_13p6TeV    : 19982.5
+WtoLNu-2Jets_13p6TeV               : 64481.58
+WW_13p6TeV                         : 125.7
+TTto2L2Nu_13p6TeV                  : 98.0
+TbarWplusto2L2Nu_13p6TeV           : 4.67
+TWminusto2L2Nu_13p6TeV             : 4.67
+TTHtoNon2B_13p6TeV                 : 0.24111
+TTLL_MLL-4to50_13p6TeV             : 0.03949
+TTLL_MLL-50_13p6TeV                : 0.08646
+WWW_13p6TeV                        : 0.23280
+WZto3LNu_13p6TeV                   : 5.31
 
 #Numbers from SMP-24-005
 VH_HtoNonbb_13p6TeV                    : 0.9561
 WZZ_13p6TeV                            : 0.06
 ZZZ_13p6TeV                            : 0.02
+GluGluToContinto2Zto2E2Mu_13p6TeV      : 0.0061150
+GluGluToContinto2Zto2E2Tau_13p6TeV     : 0.0061150
+GluGluToContinto2Zto2Mu2Tau_13p6TeV    : 0.0061150
+GluGlutoContinto2Zto4Mu_13p6TeV        : 0.003
+GluGlutoContinto2Zto4E_13p6TeV         : 0.003
+GluGlutoContinto2Zto4Tau_13p6TeV       : 0.003
+ZZto4L_13p6TeV                         : 1.65
+WWto2L2Nu_13p6TeV                      : 12.98
+WWZ_13p6TeV                            : 0.19
+
+#XSDB
+TWZToLL_tlept_Wlept_13p6TeV            : 0.001669
+TTZToQQ_13p6TeV                        : 0.6603
+WWZ4l_13p6TeV                          : 0.002244
+
+#GenXSecAnalyzer
+ZHToHtoWWto2L2Nu_13p6TeV         : 0.0195
+GluGluZHToHtoWWto2L2Nu_13p6TeV   : 0.0033
```

Comment:
And these.

Reply:
But based on what you and John explained, these are _probably_ ok, right? I.e. they should just match what's in the current master?

Though, since it seems there is a lot of confusion about what happened and what was intentional or not, it might be good for someone to just clone to copies of the repo, and on one check out the master branch and on the other check out this branch. Then just for a sanity check, just do a diff between all of the files that the modified files listed in the "Files changed" tab of the PR (just to make sure they match what's in the master). And then report the results here. E.g. showing:
```
diff path/to/master/copy/file1.py path/to/this/branch/file1.py
diff path/to/master/copy/file2.py path/to/this/branch/file2.py
```
For all of the "modified" files. Hopefully there is no output from that (which would confirm that indeed we are not introducing any unintentional changes from the master). 

Could one of the proponents of the PR do this and post the results here?

Reply:
Nevermind, Brent's solution below is much cleaner than what I was proposing here.